aggregator/src/main/scala/ai/chronon/aggregator/row/RowAggregator.scala

```diff
@@ -70,6 +70,11 @@ class RowAggregator(val inputSchema: Seq[(String, DataType)], val aggregationPar
     .toArray
     .zip(columnAggregators.map(_.irType))
 
+  val incSchema = aggregationParts
```

Comment:
I am thinking that we should use a full `incremental` in the code and we can keep `inc` as the suffix for the table so the table names are not getting too long. What do you think? 
```suggestion
  val incrementalSchema = aggregationParts
```

Reply:
yes. good to use full name.